pkg/rbac/tokenReview_test.go

```
@@ -94,6 +96,68 @@ func Test_IsValidToken_expiredCache(t *testing.T) {
+		key := hashToken(fmt.Sprintf("existing-token-%d", i))
+		mock_cache.tokenReviews[key] = &tokenReviewCache{
+			authClient: mock_cache.authnClient,
+			meta:       cacheMetadata{updatedAt: time.Now()},
+			tokenReview: &authv1.TokenReview{
+				Status: authv1.TokenReviewStatus{Authenticated: true},
+			},
+		}
+	}
+
+	// Adding one more token should not grow the map beyond the cap.
+	mock_cache.GetTokenReview(context.TODO(), "overflow-token")
```

golangci-lint (2.12.2)

[error] Error return value of `mock_cache.GetTokenReview` is not checked

(errcheck)